www/form/inner.js

```diff
@@ -4020,11 +4020,9 @@ define([
             var changeType;
             if (editable) {
                 // Drag handle
-                dragHandle = h('span.cp-form-block-drag-handle', [
-                    h('i.fa.fa-ellipsis-h'),
-                    h('i.fa.fa-ellipsis-h'),
-                ]);
-
+                let drag = $('#toggle-drag-off').attr('class').indexOf('toggle-active') !== -1 ? false : true;
+                var dragEllipses = drag ? [h('i.fa.fa-ellipsis-h'), h('i.fa.fa-ellipsis-h')] : undefined;
```

Comment:
Removing the drag handle from DOM when drag is disabled means you'll need to redraw the entire page with all the questions whenever the user switches the mode. On mobile, it could take an unnecessarily "long" time for this.
A better way to improve performances on mobile could be to add both the drag handle and the arrows to the DOM, but add `display: none` to the one that shouldn't be visible.
You'd only have to update a class of a top-level container to switch which action is visible on all the questions at once, without redrawing the page.